### PR TITLE
Copy REPL content without prompts for plain text clipboard formats

### PR DESCRIPTION
**Pull request by cston**

### CODE REVIEW

**Comment by cston** [comment]
@tmat, @amcasey, @kevinh-ms, @ManishJayaswal

File: src/InteractiveWindow/Editor/InteractiveWindow.cs [hunk 1]
```diff
@@ -734,6 +737,55 @@ private void CutOrDelete(IEnumerable<SnapshotSpan> projectionSpans, bool isCut)
...
+        /// <summary>
+        /// Copy the entire selection to the clipboard for RTF format and
+        /// copy the selection minus any prompt text for other formats.
+        /// That allows paste into code editors of just the code and
+        /// paste of the entire content for editors that support RTF.
+        /// </summary>
+        private void CopySelection()
+        {
+            var spans = _textView.Selection.SelectedSpans;
+            var text = spans.Aggregate(new StringBuilder(), GetTextWithoutPrompts, b => b.ToString());
```
Comment (amcasey) [inline src/InteractiveWindow/Editor/InteractiveWindow.cs]: Is ```PooledStringBuilder``` available here?

File: src/InteractiveWindow/Editor/InteractiveWindow.cs [hunk 2]
```diff
@@ -734,6 +737,55 @@ private void CutOrDelete(IEnumerable<SnapshotSpan> projectionSpans, bool isCut)
...
 
+        /// <summary>
+        /// Copy the entire selection to the clipboard for RTF format and
+        /// copy the selection minus any prompt text for other formats.
+        /// That allows paste into code editors of just the code and
+        /// paste of the entire content for editors that support RTF.
+        /// </summary>
+        private void CopySelection()
+        {
+            var spans = _textView.Selection.SelectedSpans;
```
Comment (amcasey) [inline src/InteractiveWindow/Editor/InteractiveWindow.cs]: Does this expand to the current line if the (user's) selection is empty?

Comment (cston) [inline src/InteractiveWindow/Editor/InteractiveWindow.cs]: No, I don't believe so. I still need to handle line selection.

**Comment by amcasey** [comment]
Is this testable?

**Comment by amcasey** [comment]
:+1:

File: src/InteractiveWindow/Editor/InteractiveWindow.cs (hunk 1 above)
Comment (cston) [inline src/InteractiveWindow/Editor/InteractiveWindow.cs]: CopySelection is only called once for a user action so the allocation won't be an issue.

**Pull request merged**